charts/request-vm-cnv/INSTALL.md

```diff
@@ -71,11 +71,79 @@ You can patch the resource by running (update it if needed with your own values)
 
 Run the following to set K_SINK and MOVE2KUBE_URL environment variables in the workflow:
 ```console
-oc -n ${TARGET_NS} patch sonataflow request-vm-cnv --type merge -p '{"spec": { "podTemplate": { "container": { "env": [{"name": "JIRA_URL", "value": "<Jira URL>"}, {"name": "JIRA_USERNAME", "value": "<Your jira username>"}, {"name": "JIRA_API_TOKEN", "value": "<Your jira token>"}, {"name": "OCP_API_SERVER_URL", "value": "<Target openshift URL>"}, {"name": "OCP_API_SERVER_TOKEN", "value": "<Token to access the target OCP cluster>"}]}}}}'
+oc -n sonataflow-infra patch sonataflow modify-vm-resources --type merge -p '{
+  "spec": {
+    "podTemplate": {
+      "container": {
+        "env": [
+          {
+            "name": "JIRA_URL",
+            "value": "<jira url>"
+          },
+          {
+            "name": "JIRA_USERNAME",
+            "value": "<jira username>"
+          },
+          {
+            "name": "JIRA_API_TOKEN",
+            "value": "<jira token>"
+          },
+          {
+            "name": "OCP_API_SERVER_URL",
+            "value": "<OCP API URL>"
+          },
+          {
+            "name": "OCP_API_SERVER_TOKEN",
+            "value": "<OCP token>"
+          },
+          {
+            "name": "OCP_CONSOLE_URL",
+            "value": "<OCP console URL>"
+          }
+        ]
+      }
+    }
+  }
+}
+'
 ```
 Ie:
 ```console
-oc -n ${TARGET_NS} patch sonataflow request-vm-cnv --type merge -p '{"spec":  { "podTemplate": { "container": { "env": [{"name": "JIRA_URL", "value": "https://masayag.atlassian.net"}, {"name": "JIRA_USERNAME", "value": "=moti.asayag@gmail.com"}, {"name": "JIRA_API_TOKEN", "value": "<Your jira token>"}, {"name": "OCP_API_SERVER_URL", "value": "https://api.cluster-ddf9q.dynamic.redhatworkshops.io:6443"}, {"name": "OCP_API_SERVER_TOKEN", "value": "<Token to access the target OCP cluster>"}]}}}}'
+oc -n sonataflow-infra patch sonataflow modify-vm-resources --type merge -p '{
+  "spec": {
+    "podTemplate": {
+      "container": {
+        "env": [
+          {
+            "name": "JIRA_URL",
+            "value": "https://masayag.atlassian.net"
+          },
+          {
+            "name": "JIRA_USERNAME",
+            "value": "moti.asayag@gmail.com"
+          },
+          {
+            "name": "JIRA_API_TOKEN",
+            "value": "<token>"
+          },
+          {
+            "name": "OCP_API_SERVER_URL",
+            "value": "https://api.cluster-hz2vh.dynamic.redhatworkshops.io:6443"
```

Comment:
before merging this, can you remove all those links with just example links and not real ones?

Reply:
sure, done